common/url_test.go

```diff
@@ -421,3 +421,11 @@ func TestCompareURLEqualFunc(t *testing.T) {
 func CustomCompareURLEqual(l *URL, r *URL, execludeParam ...string) bool {
 	return l.PrimitiveURL == r.PrimitiveURL
 }
+
+func TestURLWithLowerVersion(t *testing.T) {
```

Comment:
I think your test case has not cover all scene.
You should provide a new test case about `default.version` replace `version`.